src/nimiSlides.nim

```diff
@@ -541,15 +541,24 @@ template align*(text: string, body: untyped) =
   body
   nbRawHtml: "</div>"
 
+#templates can't have default args and untyped args at the same time
+#so we use overloading to get the same effect
+
+template columns*(columnGap: float, body: untyped) =
+  #tempted to use fmt"", but strformat doesn't support template args in the format string
```

Comment:
In these cases I usually use the `"$1 $2" % [x, y]` from `strutils` to do string interpolation. It's easier to read than concatenating strings IMO.